qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -107,17 +107,24 @@ def __init__(self, provider: 'ibm_provider.IBMProvider') -> None:
         self._ws_url = provider.credentials.runtime_url.replace('https', 'wss')
         self._programs = {}  # type: Dict
 
-    def pprint_programs(self, refresh: bool = False) -> None:
+    def pprint_programs(self, refresh: bool = False, include_details: bool = False) -> None:
         """Pretty print information about available runtime programs.
 
         Args:
             refresh: If ``True``, re-query the server for the programs. Otherwise
                 return the cached value.
+            include_details: If ``True`` print all details about available runtime programs
+                including version, creation date, maximum execution time, and input parameters.
```

Comment:
```suggestion
            include_details: If ``True`` print all details about available runtime programs.
```
Let's not list all things since a) it's missing interim results / return values and b) we'd have to update every time schema changes.